src/components/Events.tsx

```diff
@@ -0,0 +1,79 @@
+import Link from 'next/link'
+import React from 'react'
+import type {EventData} from '../types/index'
+import DisplayRC from './DisplayRC'
+
+const events: EventData[] = [
+    {
+        "id": 1,
+        "title": "Physical Meetups",
+        "description": "Physical meetups where we share, learn and network.",
+        "image_url": "https://bit.ly/placeholder-img",
```

Comment:
We should load the images provided in the `events.json`. However, I think you will have issues loading the images from Google Photos/Drive. I suggest, for all the URLs that link to Google Photos/Drive, you open the image in the browser, download it, rename it to a short descriptive name and save it in the `public/images` folder. Then we can load it from there.

Then this property would be `image_url: '/images/physical_meetups.jpg',`, for example.

You mentioned Cloudinary, we can consider that in future, but I think it is OK to load the current 6 images locally.